src/cljs/velho_ds/molecules/field.cljs

```diff
@@ -145,127 +167,127 @@
 (defn multiline-field [properties]
   (create-input-field properties :textarea))
 
-
 (defn dropdown-menu [{:keys [label
                              placeholder
                              item-list
-                             on-change-fn
-                             on-blur-fn
-                             on-focus-fn
-                             on-item-select-fn
-                             selected-item
-                             icon-click-fn
-                             icon
-                             error-messages
+                             selected-fn
+                             preselected-value
                              disabled
                              styles]}]
-  (let [item-list-keys (atom [])
-        state (r/atom {:id (subs (str (rand)) 2 9)
-                       :input-text (:label selected-item)
-                       :selected-item (if selected-item [selected-item] [])
-                       :selected-idx nil
-                       :items []
-                       :item-count (apply + (map #(count (:items %)) item-list))
-                       :focus false
-                       :disabled (if disabled disabled false)})
+  (assert (fn? selected-fn) ":selected-fn function is required for dropdown-menu")
+  (assert (vector? item-list) ":item-list vector is required for dropdown-menu")
```

Comment:
(assert (or (nil? preselected-values) (vector? preselected-values)) ":preselected-values must be vector when given")